message.go

```diff
@@ -19,6 +19,8 @@ const (
 	CompressionZSTD   CompressionCodec = 4
 )
 
+const timestampTypeMask = 0x08
+
```

Comment:
Move this below the `CompressionCodec.String` function and `CompressionLevelDefault` const.